site/docs/components/drawer/examples.mdx

```diff
@@ -8,35 +8,47 @@ data:
 ---
 
 <LivePreviewControls>
-<LivePreview componentName="drawer" exampleName="TopPositioned" >
+<LivePreview componentName="drawer" exampleName="DefaultPrimary" >
 
-### Top positioned
+### Default primary
 
-Set `position="top"` to display the drawer component at the top.
+Drawers are by default, anchored to the `left` edge of the screen. You can also place them on the `right`, `top` and `bottom` edges using `position` prop.
 
 </LivePreview>
 
-<LivePreview componentName="drawer" exampleName="RightPositioned" >
+<LivePreview componentName="drawer" exampleName="DefaultSecondary" >
 
-### Right positioned
+### Default secondary
+
+Set `variant="secondary"` for secondary drawer.
+
+</LivePreview>
+
+<LivePreview componentName="drawer" exampleName="RightWithFormField" >
```

Comment:
Why is calling out FormField usage relevant to drawer?

Reply:
Updated to RightUseCase instead. Examples currently include: Default, Position, RightUseCase (pending design mockup), TopUseCase  (pending design mockup), BottomUseCase (pending design mockup), OptionalCloseButton.

Reply:
please do share if another name fits better! @joshwooding